wocg/main.py

```diff
@@ -80,23 +72,26 @@ def main():
         addons_dir_path = os.path.join(
             svn_dir, project.slug, main_component.slug, addons_dir)
         existing_components_slug = _get_all_components_slug(project)
-        addons = get_translatable_addons(addons_dirs=[addons_dir_path])
+        addons = get_translatable_addons(addons_dir_path)
         main_filemask = main_component.filemask
-        for addon, addon_dir in addons.items():
-            addon_component_name = _get_component_name(project, addon)
-            addon_component_slug = _get_component_slug(project, addon)
+        main_new_base = main_component.new_base
+        for addon in addons.keys():
+            addon_component_name = get_component_name(project, addon)
+            addon_component_slug = get_component_slug(project, addon)
             if addon_component_slug in existing_components_slug:
                 logger.info('component already exist for addon %s : %s' % (
                     addon, addon_component_slug))
                 continue
             logger.info('Begin generation for addon %s' % addon)
             filemask = main_filemask.replace(main_component_addon_name, addon)
+            base_file = main_new_base.replace(main_component_addon_name, addon)
             logger.info('New filemask %s' % filemask)
             new_component = main_component
             new_component.pk = None
             new_component.name = addon_component_name
             new_component.slug = addon_component_slug
             new_component.filemask = filemask
+            new_component.new_base = base_file
             new_component.repo = repo
             new_component.save()
```

Comment:
unset *Exported repository URL* and *Repository push URL* (they are forbidden for a linked component)